### ISSUE TITLE
[Feature Request] Early stopping?

### ISSUE THREAD

**Issue by deklanw** [sisinflab/elliot#5]
Not seeing an option for early stopping based on validation metric performance. Running neural models with fixed numbers of epochs is either time-consuming (treating epoch number as a hyperparameter) or risks under/overfitting (fixing the epochs).
